sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs

```diff
@@ -438,8 +440,9 @@ private async Task ApplyPlugins(IReadOnlyList<ServiceBusReceivedMessage> message
             int maxMessages,
             CancellationToken cancellationToken)
         {
-            Argument.AssertNotDisposed(IsClosed, nameof(ServiceBusReceiver));
             Argument.AssertAtLeast(maxMessages, 1, nameof(maxMessages));
+            Argument.AssertNotDisposed(IsClosed, nameof(ServiceBusReceiver));
```

Comment:
I reordered the assertions to match other call sites and consistently validate the parameters before the receiver closed state and connection.   I'm not sure if the ordering was intentionally inconsistent, though;  if so, please let me know and I'll restore the original ordering.